src/platform.cpp

```diff
@@ -0,0 +1,18 @@
+// Copyright © 2024 Dylan Baker
+// SPDX-License-Identifier: MIT
+
+#include "platform.hpp"
+
+namespace fs = std::filesystem;
+
+namespace platform {
+
+    const fs::path libdir() {
+        // TODO: libdir needs to be configurable based on the personality,
+        //       and different name schemes.
+        //       This is complicated by the fact that different distros have
+        //       different schemes.
+        return "lib";
```

Comment:
Maybe an environment variable and/or a CLI flag is in order? Like `CPS_LIBDIR`?

Reply:
Yeah, I think it makes sense for a first pass at least to have an option to configure this, or to make it == to the libdir that the project is configured with. I actually kinda like that second idea as a first pass

Reply:
I went ahead and changed this to use Meson's `libdir` built-in option as a default, which is equivalent to CMake's `CMAKE_INSTALL_LIBDIR` option. It makes sense to me that cps-config would search it's own libdir for cps files